config.json

```diff
@@ -19,7 +19,8 @@
       "%{kebab_slug}.ys"
     ],
     "test": [
-      "%{kebab_slug}/test/test-1.ys"
+      "test/test-1.ys",
+      "Makefile"
```

Comment:
What does this line do?

Reply:
The `"Makefile"` line...

Reply:
When you use the `configlet create` command, it will touch this file in the directory for the new exercise being created. It will also include the `Makefile` in the exercise's `config.json` file, so that it is correctly identified as a file used for running tests. (If it is not included, the online editor may identify it as a file that should not be present.)

Reply:
Gotcha. I can adjust for that. Thanks.